ddtrace/internal/dm.py

```diff
@@ -0,0 +1,100 @@
+import enum
```

Comment:
Can we rename this module to something a little more descriptive. From a glance it's hard to know what `dm` is